group_vars/all.yml.sample

```diff
@@ -768,6 +768,8 @@ dummy:
 # We only need this for SSL (https) connections
 #grafana_crt: ''
 #grafana_key: ''
+# Server hostname for which grafana_crt is valid
+#grafana_server_addr: ''
```

Comment:
2 things:

1/ you shouldn't edit this file directly.
please, edit `roles/ceph-defaults/defaults/main.yml` and run `./generate_group_vars_sample.sh`

2/ That will work as it is, but I wonder whether we would better setting a default value here, then we would have a condition like this on the tasks you've modified below:  

```yaml
  when:
    - grafana_server_addr != "whatever-default-value"
```